vue-cli-2/src/components/Card.cy.js

```diff
@@ -0,0 +1,36 @@
+import { mount } from "@cypress/vue";
+import Card from "./Card.vue";
+import Vuetify from "vuetify";
+
+describe("CustomCard.vue", () => {
+  let vuetify;
+
+  beforeEach(() => {
+    vuetify = new Vuetify();
+  });
+
+  it("displays the correct title", () => {
+    mount(Card, {
+      propsData: {
+        title: "Hello, World!",
+      },
+    });
+    cy.get("[data-test='title']").contains("Hello, World!");
+  });
+
+  it.only("should emit an event when the action v-btn is clicked", () => {
```

Comment:
```suggestion
  it("should emit an event when the action v-btn is clicked", () => {
```
@robertguss remove this only so all tests run

Reply:
doh! removed.